docs/spec/qualifiers.rst

```diff
@@ -272,33 +283,54 @@ details of the syntax:
     SmallInt = Annotated[int, ValueRange(0, 100)]
     SmallInt(1)  # Type error
 
+:pep:`593` and an earlier version of this specification used the term
+"annotations" instead of "metadata" for the extra arguments to
+``Annotated``. The term "annotations" is deprecated to avoid confusion
+with the parameter, return, and variable annotations that are part of
+the Python syntax.
+
+Meaning
+^^^^^^^
+
+The metadata provided by ``Annotated`` can be used for either static
+or runtime analysis. If a library (or tool) encounters an instance of
+``Annotated[T, x]`` and has no special logic for metadata element ``x``, it
+should ignore it and treat the expression as equivalent to ``T``. Thus, in general,
+any :term:`type expression` or :term:`annotation expression` may be
+wrapped in ``Annotated`` without changing the meaning of the
+wrapped expression. However, type
+checkers may additionally choose to recognize particular metadata elements and use
+them to implement extensions to the standard type system.
+
+``Annotated`` metadata may apply either to the base expression or to the symbol
+being annotated, or even to some other aspect of the program.
 
-Consuming annotations
-^^^^^^^^^^^^^^^^^^^^^
+Consuming metadata
+^^^^^^^^^^^^^^^^^^
 
-Ultimately, the responsibility of how to interpret the annotations (if
+Ultimately, the responsibility of how to interpret the metadata (if
```

Comment:
pre-existing wording nit: redundant and awkward "responsibility of" here:
```suggestion
Ultimately, how to interpret the metadata (if
```

Reply:
That reads awkward to me too, how about:
```suggestion
Ultimately, deciding how to interpret the metadata (if
```

Reply:
Sure, that's fine.